docs/requirements.txt

```diff
@@ -35,6 +35,7 @@ docutils==0.16 \
     #   -r docs/requirements.txt
     #   sphinx
     #   sphinx-rtd-theme
+    #   sphinx-tabs
```

Comment:
Nit: if you could make changes to `docs/requirements.txt` in a separate PR, it would make life slightly easier for maintainers downstream.

Reply:
@dmitri-d on this point i would like to push back - adding yaml to the docs build requirements is not needed without this PR - it makes sense to add it here i think

Reply:
the extraneous changes here are produced by `pip-compile` not sure why they hadnt been set before - but im guessing that dependabot may be to blame for at least the hash ordering